### PR TITLE
documentation: update my name (Ortmann -> Knüpfer)

### PR DESCRIPTION
**Pull request by LudwigOrtmann**

### CODE REVIEW

**Comment by OlegHahm** [comment]
Seems to be epidemic. ;)

**Comment by LudwigOrtmann** [comment]
:hear_no_evil:

**Comment by LudwigOrtmann** [comment]
Updated my git name/email and force-pushed.

File: boards/chronos/board_init.c [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * @brief       eZ430-chronos board initialization
  *
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
- * @author      Ludwig Ortmann <ludwig.ortmann@fu-berlin.de>
+ * @author      Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
```
Comment (authmillenon) [inline boards/chronos/board_init.c]: You were able to change that as an alumnus?

Comment (authmillenon) [inline boards/chronos/board_init.c]: (I mean the mail address)

Comment (LudwigOrtmann) [inline boards/chronos/board_init.c]: Yes. When I exmatriculated I entered the [Ernst-Reuter-Gesellschaft der Freunde, Förderer und Ehemaligen der Freien Universität Berlin e.V.](http://www.fu-berlin.de/sites/alumni/erg/). After the name change I updated my name there and after a while this change fluctuated into the FU system. Once that happened I was able to add a new address for my new name (I am keeping my old address as well) in the regular zedat interface.
On a related note: I was able to get my certificate in my new name without any hassle as well. (I changed names before I presented the thesis).

Comment (authmillenon) [inline boards/chronos/board_init.c]: \o/

**Pull request merged**